DEVELOPING.md

```diff
@@ -0,0 +1,36 @@
+# Code Development
+
+This markdown file serves as a developer's guide for people who are enthusiastically inquisitive and driven towards making 
+open-source contributions to [Java-Manta](http://joyent.github.com/java-manta) which is a community-maintained Java
+SDK for interacting with Joyent's Manta system. 
+
+For more general information on the using the SDK with examples, Kindly visit [Usage](https://github.com/joyent/java-manta#usage) 
+with code-snippets illustrated as basic examples. It will be periodically updated with basic instructions on how to 
+incorporate certain features within the SDK which is holistically meant towards encouraging contributions. For more details, 
+Refer [Contributing.md](https://github.com/joyent/java-manta/blob/master/CONTRIBUTING.md).
+ 
```

Comment:
Change this section from:
# Code Development

This markdown file serves as a developer's guide for people who are enthusiastically inquisitive and driven towards making 
open-source contributions to [Java-Manta](http://joyent.github.com/java-manta) which is a community-maintained Java
SDK for interacting with Joyent's Manta system. 

For more general information on the using the SDK with examples, Kindly visit [Usage](https://github.com/joyent/java-manta#usage) 
with code-snippets illustrated as basic examples. It will be periodically updated with basic instructions on how to 
incorporate certain features within the SDK which is holistically meant towards encouraging contributions. For more details, 
Refer [Contributing.md](https://github.com/joyent/java-manta/blob/master/CONTRIBUTING.md).

TO:
## Configuration Parameter Contributions
This section describes how to make contributions to the configuration parameters for [java-manta](https://github.com/joyent/java-manta/blob/master/USAGE.md#configuration). Java-manta is a community-maintained Java SDK for interacting with Joyent's Manta object storage system.
### Recommended Prerequisite References

- [Using the SDK](https://github.com/joyent/java-manta#usage)
- [Contributing to java-manta](https://github.com/joyent/java-manta/blob/master/CONTRIBUTING.md)
- [Configuration documentation](https://github.com/joyent/java-manta/blob/master/USAGE.md#configuration)